ibm_translation_to_chinese.Rmd

```diff
@@ -8,4 +8,4 @@ https://www.ibm.com/design/language/data-visualization/overview/
 
 
 The translated version in Chinese can be accessed in:  
-https://github.com/zc2583/cc21/blob/IBM_DesignLanguage_Translated/IBM_Design_Translated.pdf  
\ No newline at end of file
+https://github.com/zc2583/cc21/blob/main/5702%20CC.pdf  
```

Comment:
Please add this fill to the resources directory or add it as an external link to google drive etc.

> Links to internal files must contain resources/<project_name>/ in the path, such as: ![Test Photo](resources/sample_project/election.jpg)
Refer here: https://jtr13.github.io/cc21/tutorial-for-pull-request-mergers.html

Reply:
Hi Smarth,

This link is actually not an internal link to the public repository. This is a link to my own branch which is  independent of the branch of pull request.
I really want to upload the file to google drive, but I have trouble making it public. So I do it in this way...
Is this ok?

Best,
Zhongsheng

Reply:
Hello Zhongsheng, 

I thought you did it by mistake. Completely fine if it is intentional. I am merging this in!

Thank you
Smarth